### DISCUSSION TITLE
I can't start the container !

### DISCUSSION THREAD

**Discussion by chitalov** [Taxel/PlexTraktSync#1478] [Support] [answered]
I am trying to run this tool on the given example but it doesn't work.
Here is my log and Docker Compose stack
![image](https://user-images.githubusercontent.com/73425550/236634582-204879d2-1fe3-43b6-b6dc-07034cd63059.png)
`ERROR    No more methods to connect. Giving up.                                 

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7fab803e8ed0>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7fab80108b10>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    No more methods to connect. Giving up.                                 

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f9ef4b52810>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f9ef4b63310>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    No more methods to connect. Giving up.                                 

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7fe1891fe5d0>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7fe18920f910>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    No more methods to connect. Giving up.                                 

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f747dc78ed0>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f747d998d50>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    No more methods to connect. Giving up.                                 

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7ff7d2910e50>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7ff7d2633190>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    No more methods to connect. Giving up.                                 

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f478173e5d0>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f478174f890>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    No more methods to connect. Giving up.                                 

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f9b49b58e50>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f9b4987b210>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    No more methods to connect. Giving up.                                 

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f0ddf5cced0>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f0ddf2ecd50>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    No more methods to connect. Giving up.                                 

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f977f2c2690>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    HTTPConnectionPool(host='localhost', port=32400): Max retries exceeded 

         with url: / (Caused by                                                 

         NewConnectionError('<urllib3.connection.HTTPConnection object at       

         0x7f977f2d0ad0>: Failed to establish a new connection: [Errno 111]     

         Connection refused'))                                                  

ERROR    No more methods to connect. Giving up.                                 
`

**Comment by chitalov** [comment]
I was able to run the script, it just doesn't mention anywhere that I have to manually enter the IP address and token in the servers.yml file.  please add this information to the main page!
![Screenshot_20230509_080331_Chrome](https://user-images.githubusercontent.com/73425550/236999024-7b39002e-a5b4-4fbd-9452-ca04b28ceb82.jpg)

**Reply by simonc56** [reply, marked as answer]
As explained in Readme, you need to run the `login` command at first run.
It will create all needed credentials files.
